homeassistant/const.py

```diff
@@ -230,6 +230,7 @@
 ATTR_DATE = 'date'
 ATTR_TIME = 'time'
 ATTR_SECONDS = 'seconds'
+ATTR_DURATION = 'duration'
```

Comment:
Don't set a global const for only one platform. You can later consolidate some platform with this attribute and add it to global.